experimental/framework/tinylicious-client/src/TinyliciousClient.ts

```diff
@@ -53,7 +54,9 @@ export class TinyliciousClientInstance {
             runtimeFactory,
             true,
         );
-        return this.getRootDataObject(container);
+
+        const rootDataObject = await this.getRootDataObject(container);
+        return new FluidContainer(container, rootDataObject);
```

Comment:
Why is this not using the static FluidContainer create?

Reply:
already mentioned in other feedback. changed as per feedback from @ChumpChief